windows-gaming-driver/src/driver/controller.rs

```diff
@@ -308,6 +330,10 @@ impl Controller {
         assert!(self.ga != State::Suspending, "trying to exit from a suspending vm?");
         assert!(self.ga != State::Suspended, "trying to exit from a suspended vm?");
         self.ask_clipboard();
+        // if we have a temporary entry, get rid of it
+        if let Some(w) = mem::replace(&mut self.temporary_entry_writer, None) {
+            drop(w);
```

Comment:
Nothing to be unsure about: An assignment always drops the previous value (if any).